.gitlab-ci.yml

```diff
@@ -397,7 +398,7 @@ TF_Settings::Web::build:
     name: "TouchFree_Settings_Web_${env:TOUCHFREE_VERSION}_${env:CI_COMMIT_SHORT_SHA}"
     paths:
     - ./TF_Settings_Web/dist/*
-    - ./TF_Settings_Web/src-tauri/target/release/*.exe
+    - ./TF_Settings_Web/touchfree-settings-web.exe
```

Comment:
I would say just pull any .exe files from here rather than this named file specifically as we are planning on renaming

Reply:
Agree - also failed with this error 
`file 'c:\builds\apps\screen-control/TF_Settings_Web/src-tauri/target/release/touchfree-settings-web.exe' does not exist.`

Reply:
Due to the azuresigntool

Reply:
This should all be sorted now!

Reply:
Missing the entire exe now 😨